webapp/platform/components/src/generic_modal/__snapshots__/generic_modal.test.tsx.snap

```diff
@@ -54,6 +54,9 @@ exports[`GenericModal should match snapshot for base case 1`] = `
                     Close
                   </span>
                 </button>
+                <div
```

Comment:
As sanity check, does this extra div here cause any issue?

Reply:
Nope, its intended. Tested on other modals as well.